interfaces/mount/change_test.go

```diff
@@ -20,15 +20,35 @@
 package mount_test
 
 import (
+	"errors"
+
 	. "gopkg.in/check.v1"
 
 	"github.com/snapcore/snapd/interfaces/mount"
 )
 
-type changeSuite struct{}
+type changeSuite struct {
+	sys     *mount.SyscallRecorder
+	restore func()
```

Comment:
Since you suggested similar thing to me today (and I kind like that) ;), let's use BaseTest and get rid of restore before it grows

Reply:
Ack